config/detekt/detekt-baseline.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<SmellBaseline>
```

Comment:
Empty baseline, in case you want to add exceptions that should be ignored you can use `detektAllBaseline` task